services/QuillLMS/app/assets/stylesheets/pages/premium_hub/growth-diagnostic-reports.scss

```diff
@@ -0,0 +1,74 @@
+.diagnostic-reports-main {
+  max-width: calc(100% - 317px) !important;
```

Comment:
Maybe throw in a comment about where the 317px is coming from?

Reply:
I'll add a comment